inc/class-bento-events-controller.php

```
@@ -287,6 +410,70 @@ public static function load_events_controllers() {
+		 * @param array $details The event details to sanitize.
+		 * @return string Sanitized details summary for logging.
+		 */
+		private static function sanitize_details_for_logging( $details ) {
+			if ( empty( $details ) || ! is_array( $details ) ) {
+				return '[no-details]';
+			}
+			
+			$safe_keys = array();
+			$sensitive_patterns = array( 'email', 'phone', 'address', 'name', 'first_name', 'last_name', 'user_login' );
+			
+			foreach ( $details as $key => $value ) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$value'. (Unused Code Rules)

(UnusedLocalVariable)